stdlib/ballerina-http/src/main/ballerina/ballerina/net/http/commons.bal

```diff
@@ -2,24 +2,14 @@ package ballerina.net.http;
 
 import ballerina.mime;
 
-function getHeadersFromEntity (mime:Entity entity, string headerName)(mime:HeaderValue[]) {
-    if (entity.headers == null) {
-        return null;
-    }
-    var headerValues = entity.headers[headerName];
-    if (headerValues == null) {
-        return null;
-    }
-    return getHeaderValueArray(headerValues, headerName);
+function getFirstHeaderFromEntity(mime:Entity entity, string headerName)(string) {
+    var headerValue, _ = (string[]) entity.headers[headerName];
+    return headerValue == null ? null : headerValue[0];
 }
 
-function getHeaderValueArray (any headerValues, string headerName)(mime:HeaderValue[]) {
-    var valueArray, err = (mime:HeaderValue[]) headerValues;
-    if (err != null) {
-        error errMsg = {msg:"expect 'ballerina.net.mime:HeaderValue[]' as header value type of : " + headerName};
-        throw errMsg;
-    }
-    return valueArray;
+function getHeadersFromEntity (mime:Entity entity, string headerName)(string[]) {
```

Comment:
Do we need the spacing between the function name and "("  ?

Reply:
Removed the space

Reply:
In other ballerina methods, There is a space between name and "(". I changed these functions too